client/src/components/advancedSearch/CopyButton.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { useState, FC } from 'react';
+import PropTypes from 'prop-types';
+import {
+  makeStyles,
+  Theme,
+  createStyles,
+  Tooltip,
+  IconButton,
+  Fade,
+} from '@material-ui/core';
+
+interface CopyButtonProps {
+  className?: string;
+  icon?: any;
+  label: string;
+  value: string;
+  others?: any;
+}
+
+const CopyIcon = (
```

Comment:
We have icons component, so we usually put icon in assets/icons and import to icons component.
It's easy for us to know what icons we have